docs/user/installation.rst

```diff
@@ -228,21 +228,21 @@ To check whether it is found by Theano, run the following command:
 It will print ``True`` if everything is fine, or an error message otherwise.
 There are no additional steps required for Theano to make use of cuDNN.
 
-Docker images
-=============
+Docker
+======
 
-Instead of manually installing Theano and Lasagne on your machines as described
-above, you may want to use a pre-made `Docker <https://www.docker.com/whatisdocker>`_
-image: `Lasagne Docker (CPU) <https://hub.docker.com/r/kaixhin/lasagne/>`_ or
+Builds of Lasagne are available as `Docker <https://www.docker.com>`_
+images: `Lasagne Docker (CPU) <https://hub.docker.com/r/kaixhin/lasagne/>`_ or
```

Comment:
> Now that you mention it the reason must be that I literally took your wording from #411.

Ah, that must be why I liked the original wording better ;)

> Would you like to have the wording changed back or not? I can just amend this commit to keep it tidy.

Yes, amending the comment to restore the original formulation would be appreciated! Also I think the direct link to "what is docker" is more useful for people unfamiliar with it than just linking to docker.com. The latter is easy to find from the former, but not vice versa!